packages/cli/src/commands/env/index.ts

```diff
@@ -69,31 +72,31 @@ export default async function main(client: Client) {
         printHelp(listSubcommand);
         return 2;
       }
-      telemetry.trackCliSubcommandList(subcommand);
+      telemetry.trackCliSubcommandList(subcommandOriginal);
       return ls(client, args);
     case 'add':
       if (needHelp) {
         telemetry.trackCliFlagHelp('env', 'add');
```

Comment:
I think we should audit and do in these help functions as well:

```suggestion
        telemetry.trackCliFlagHelp('env', subcommandOriginal);
```

Reply:
I want to do these on a separate PR.

Reply:
https://github.com/vercel/vercel/pull/12552